src/views/ContinuationIn/ContinuationInBusinessBc.vue

```diff
@@ -142,15 +156,17 @@ export default class ContinuationInBusinessBc extends Mixins(CommonMixin) {
   readonly validComponents = [
     'office-address-header',
     'registered-office-contact-header',
-    'folio-reference-number-header'
+    'folio-reference-number-header',
+    'name-translation-header'
   ]
 
   /** Object of valid flags. Must match validComponents. */
   get validFlags (): object {
     return {
       validAddressForm: this.addressFormValid,
       validBusinessContactForm: this.businessContactFormValid,
-      validFolioReferenceNumber: !this.isPremiumAccount || this.getFolioNumberValid
+      validFolioReferenceNumber: !this.isPremiumAccount || this.getFolioNumberValid,
+      validNameTranslation: this.getNameTranslationsValid
```

Comment:
Very nice following the existing patterns 👍

Reply:
Oh, darn. This won't work because `name-translation-header` isn't an id on this page.

Also, these flags (and the list on line 156) should be in the order of the components on the page. These are used by `validateAndScroll()` to scroll to the first invalid component.

Reply:
I think it would be OK to scroll to line 5 (change "company-name" to "company-name-header"). And also rename the variable on line 169 accordingly.

Reply:
Updated, thank you. Now it works